main/openssh/APKBUILD

```diff
@@ -39,32 +39,22 @@ source="http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/$pkgname-$_myver.tar
 
 # HPN patches are from: http://www.psc.edu/index.php/hpn-ssh
 
-builddir="$srcdir"/$pkgname-$_myver
-prepare() {
-	cd "$builddir"
-	default_prepare
-	sed -i -e '/_PATH_XAUTH/s:/usr/X11R6/bin/xauth:/usr/bin/xauth:' \
-		pathnames.h || return 1
-}
-
 build() {
 	cd "$builddir"
-	export LD="$CC"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
 		--sysconfdir=/etc/ssh \
-		--datadir=/usr/share/openssh \
```

Comment:
I remove it because there is no datadir in this package. I think it appears as a copy-paste result from another APKBUILD.